City-of-Pori/pori-kada
# PORI-395 Carousel layout bug

…th touchevents

To test:

1. drush cc all
2. Use the chrome inspector and pick a mobile phone from the select list. For example Pixel 2 and make sure the carousel works correctly.
